ui/src/components/Main.jsx

```diff
@@ -75,6 +90,7 @@ class Main extends React.Component {
   }
 
   render() {
+    setBadgeCount(this.props.remoteAccess.chatMessageCount);
```

Comment:
the chat widget (`react-chat-widget`) now apparently handles the message count internally (see setBadgeCount, https://github.com/Wolox/react-chat-widget) so we need to override that behavior

Reply:
This should be called in `componentDidUpdate` (or `useEffect` if it was a function component).

If you like, I can open a sub-PR to refactor `Main` into a function component (I had already done so yesterday actually, but hadn't pushed it yet).

Reply:
Actually, that for some reason did not work. I had to move `setBadeCount` to render to make it work. Wonderful for the functional component, I was hesitating of doing it as well. Ill let you continue if you already started :)

Reply:
Ill quickly address some of your comments below and then you can go for it

Reply:
Should be ok now :)

Reply:
Thanks, I'll take over.